docker/ci/dockerfiles/current/release.centos.clients.x64.arm64.dockerfile

```diff
@@ -85,7 +85,20 @@ RUN ln -sfn /usr/local/bin/python3.7 /usr/bin/python3 && \
     pip3 install pipenv && pipenv --version && pip install --upgrade pip
 
 # Install twine
-RUN pip3 install twine
+RUN pip3 install twine cmake==3.24.1.1
+
+# Install openssl1.1.1
+ENV LD_LIBRARY_PATH=$LD_LIBRARY_PATH:/usr/local/lib:/usr/local/lib64:/usr/lib
+RUN yum install -y curl libcurl-devel libfaketime perl-core pcre-devel && yum remove -y openssl-devel && yum clean all && \
```

Comment:
We are installing `openssl-devel` on line 30 and then removing here. Is that intended?

Reply:
That is exactly intended as you need openssl-devel (1.0.2) for Python37 compilation.
Only when python37 compiled and installed then pip3 can be used for cmake3 installation.
Only when cmake3 installed you can then compile openssl 1.1.1 (while also removing the old 1.0.2), in order for the osslsigncode 2.5 to compile correctly.

Thanks.

Reply:
Got it. Thanks!

Reply:
All of this hassle is due to CentOS7 ship with openssl 1.0.2, not 1.1.1.